src/script.js

```diff
@@ -1,19 +1,51 @@
 'use strict';
 
+// import {getTopRatedFilms} from '../src/services/api.js';
+
 function createLayout(content) {
```

Comment:
@Mmarvinn For me `renderPageLayout` will be better name

Reply:
thx )
fixed